src/content/en/fundamentals/performance/speed-tools/index.md

```diff
@@ -0,0 +1,334 @@
+project_path: /web/fundamentals/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+
+{# wf_updated_on: 2018-02-26 #}
+{# wf_published_on: 2018-02-26 #}
+{# wf_blink_components: N/A #}
+
+<style>
+  .text-and-image{
+    margin: 0 0 20px;
+  }
+
+  .download{
+    background: #4285f4;
+    padding: 20px;
+    color: #fff;
+    display: block;
+    height: auto;
+  }
+
+  .download:hover,
+  .download:active{
+    background: #4285f4;
+    color: #fff;
+  }
+
+  .text-and-image + .text-and-image{
+    margin-top: 40px;
+  }
+
+  .text-and-image > img,
+  .tool img{
+    display: block;
+    margin: 0 auto 20px;
+    min-width: 128px;
+    max-width: 33.33%;
+  }
+
+  .tool img{
+    margin-bottom: 0;
+    max-width: 25%;
+    min-width: 112;
+  }
+
+  .image-container{
+    padding: 20px 0;
+    margin: 0 0 20px;
+    background: #4285f4;
+  }
+
+  .text-and-image > div{
+    margin: 0;
+  }
+
+  .text-and-image > div > h3,
+  .text-and-image > div > p{
+    margin: 0;
+  }
+
+  .text-and-image > div > h4{
+    margin-top: 0;
+  }
+
+  .tool{
+    text-align: center;
+    margin: 0 0 20px;
+  }
+
+  @media screen and (min-width: 576px){
+    .text-and-image{
+      display: flex;
+      align-items: flex-start;
+      justify-content: space-between;
+    }
+
+    .text-and-image > img{
+      width: 128px;
+      margin: 0;
+    }
+
+    .text-and-image > div{
+      width: calc(100% - 144px);
+    }
+
+    .tools{
+      display: flex;
+      align-items: stretch;
+      flex-flow: row wrap;
+      justify-content: center;
+    }
+
+    .tool{
+      width: 50%;
+    }
+
+    .tool > p{
+      padding: 0 10px;
+    }
+
+    .image-container{
+      max-width: 97.5%;
+    }
+  }
+
+  @media screen and (min-width: 721px){
+    .text-and-image{
+      display: block;
+    }
+
+    .text-and-image > img{
+      margin: 0 auto 20px;
+      min-width: 128px;
+      max-width: 33.33%;
+    }
+
+    .text-and-image > div{
+      width: auto;
+    }
+
+    .tools{
+      display: block;
+    }
+
+    .tool{
+      width: 100%;
+    }
+
+    .tool > p{
+      padding: 0;
+    }
+
+    .image-container{
+      max-width: none;
+    }
+  }
+
+  @media screen and (min-width: 800px){
+    .text-and-image{
+      display: flex;
+    }
+
+    .text-and-image > div{
+      width: calc(100% - 144px);
+    }
+
+    .text-and-image > img{
+      margin: 0;
+    }
+
+    .tools{
+      display: flex;
+    }
+
+    .tool{
+      width: 50%;
+    }
+
+    .tool > p{
+      padding: 0 10px;
+    }
+
+    .image-container{
+      max-width: 97.5%;
+    }
+  }
+</style>
+
+# How To Think About Speed Tools {: .page-title }
+
+## Common myths about performance
+
+<div class="text-and-image">
```

Comment:
Is it possible to stick with primarily markdown? Using HTML & Markdown makes future maintenance much more of a challenge.